features/meta_data_scenario.feature

```diff
@@ -0,0 +1,7 @@
+Feature: Android support
+
+Scenario: Test handled Android Exception
```

Comment:
The scenario name should state what's actually being tested here, e.g. 
"Add metadata in a notify() callback". It also reveals what other cases might be needed here, like changing or removing metadata.